driver/src/main/java/org/neo4j/driver/internal/logging/JULogger.java

```diff
@@ -26,21 +26,22 @@
 public class JULogger implements Logger
 {
     private final java.util.logging.Logger delegate;
-    private final boolean debugEnabled;
 
     public JULogger( String name )
     {
         delegate = java.util.logging.Logger.getLogger( name );
-        debugEnabled = delegate.isLoggable( Level.FINE );
```

Comment:
Fair enough. We should re-visit this later on then, not doing this check is *very* expensive, each debug log call effectively creates synchronization, this was a big bottleneck in the neo4j server a few years back.